### PR TITLE
Make participant create wizard update user correctly

### LINKED ISSUE #2094: `Add new participant` wizard displays wrong data
- Go to `/<meeting-id>/participants/new`
- enter some user data which does not exist yet
- you will be directly send to step 3
- click "Back" to go to step 1
- enter username of an existing user
- select the existing user in step 2
- you will still see the old data in step 3 and not the data of the user you selected

The same happens if you again enter not-existing user data, the form in step 3 does not update to the new data.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2094

### CODE REVIEW

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/site/modules/user-components/components/user-detail-view/user-detail-view.component.ts [hunk 1]
```diff
@@ -240,13 +244,13 @@ export class UserDetailViewComponent extends BaseUiComponent implements OnInit,
...
-            this.personalInfoForm.get(formControlName)!.enable();
+            setTimeout(() => this.personalInfoForm.get(formControlName)!.enable(), 1000);
         });
 
         // Disable not permitted controls
         formControlNames.forEach(formControlName => {
             if (!fn(formControlName)) {
-                this.personalInfoForm.get(formControlName)!.disable();
+                setTimeout(() => this.personalInfoForm.get(formControlName)!.disable(), 1000);
             }
```
[the remark this answers is not in the document]
Comment (luisa-beerboom) [reply client/src/app/site/modules/user-components/components/user-detail-view/user-detail-view.component.ts:254]: Without this code, when immediately referencing an account, the checkboxes for natural person and active/inactive will be disabled.

**Review by Elblinator — CHANGES_REQUESTED** [review]
if the existing user is selected in step 2, the new data of the selected user will be displayed. 

But after the correct data is displayed in step 3:

- click 'previous' to go to step 2
- click 'previous' to go to step 1
- change username to new user 
- select 'next' 
- the data from the previously selected account will be displayed

**Review by bastianjoel — APPROVED** [review]
This seems like a lot of workarounds. I am fine with merging this but I would prefer to just do a cleanup or maybe a rewrite of this component.

**Review by Elblinator — CHANGES_REQUESTED** [review]
Now step 3 will display the correct data.

But:

1. in step one create a new user A
2. step 3 correctly displays the data
3. back to step 1, change username to already existing user B
4. in step 2 select already existing user B
5. data will be correctly displayed in step 3
6. back to step 1, change all the data back into a non-existing user C
7. step 3 will display data correctly
8. press save
9. one of two thing can now happen: 
-  a) either a new user is created with the data from the in 4. selected user B
- b) The Errormessage: Error: _A user with the username "B" already exists._ is displayed

the data from the new user C should be saved inside a user profile

**Review by Elblinator — APPROVED** [review]

**Review by bastianjoel — APPROVED** [review]
Please create a cleanup issue for the participant create wizard.

**Pull request merged**